Handle DB errors in contacts search plugin

alikon opened the pull request:
updated to handle DB errors like #6710, #6632 and for #6591

Testing info
simulate a db error

alikon committed:
Handle DB errors in contacts search plugin

updated to handle DB errors like #6710, #6632 and for #6591

Testing info
simulate a db error

zero-24 committed:
make travis happy

@alikon this should make travis happy for https://github.com/joomla/joomla-cms/pull/6711

zero-24 committed:
Update contacts.php

alikon committed:
Merge pull request #5 from zero-24/patch-9

make travis happy

nonumber commented:
@test Success

nonumber commented:
Not sure if this should get merged with regards to discussion on https://github.com/joomla/joomla-cms/pull/6710

alikon committed:
added the generic error msg

used the generic message string "There has been an error" as suggested in #6710 the string was added in #7073
